src/data/faq.json

```diff
@@ -203,7 +203,7 @@
                         "anchor": "ios_14_beta",
                         "active": true,
                         "textblock": [
-                            "With iOS 14 Beta 4, the Exposure Notification framework is available. But this version is <a href='https://developer.apple.com/support/beta-software/' target='_blank' rel='noopener noreferrer'>still under development</a> and is not public. The Corona-Warn-App is not yet supported with iOS 14 Beta 4. See the <a href='https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-14-beta-release-notes' target='_blank' rel='noopener noreferrer'>iOS & iPadOS 14 Beta Release Notes</a>."
+                            "Even though with iOS 14 Beta 4, the Exposure Notification framework is available now, the Corona-Warn-App does not support iOS 14 Beta-Versions. The Beta-Versions are <a href='https://developer.apple.com/support/beta-software/' target='_blank' rel='noopener noreferrer'>still under development</a> and are not public. The Corona-Warn-App is not yet supported with iOS 14 Beta 4. See the <a href='https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-14-beta-release-notes' target='_blank' rel='noopener noreferrer'>iOS & iPadOS 14 Beta Release Notes</a>."
```

Comment:
One point, could you remove this: 
>The Corona-Warn-App is not yet supported with iOS 14 Beta 4.

and in German this: 
>Die Corona-Warn-App wird derzeit auf iOS 14 Beta noch nicht unterstützt.

Thanks and have a nice evening.